install/run-one.yml

```diff
@@ -0,0 +1,19 @@
+---
+## Installation playbook for one role
+
+# export the role you want to run in the ‘ROLE’ environment variable
+# and run the playbook.
```

Comment:
Are there roles with dependencies which are no longer handled because the logic
is in the main playbook? (eg imapproxy.)

Is it different than something like `ansible-playbook -i ../config/hosts.yml main.yml -t $ROLETAG`?